package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "3sb-website",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "lint": "npx eslint 'site/.vitepress/**/*.{ts,vue}'",
+    "site:dev": "vitepress dev site --port 3000",
+    "site:build": "vitepress build site",
+    "site:serve": "vitepress serve site"
+  },
+  "devDependencies": {
+    "@types/glob": "^8.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.42.1",
+    "@typescript-eslint/parser": "^5.42.1",
+    "eslint": "^8.27.0",
+    "eslint-plugin-vue": "^8.7.1",
+    "glob": "^8.0.1",
```

Comment:
In theory not needed right now but will be needed for the github deploy pipeline later.